assets/messages.yml

```diff
@@ -876,6 +900,19 @@ messages:
 
         Your computer does not meet the [minimum system requirements|https://help.minecraft.net/hc/en-us/articles/360038937032-Dungeons-Minimum-Specifications-for-Gameplay] to run Minecraft Dungeons.
 
+        %quick_links%
+      fillname: []
+    - project:
+        - mclg
+      name: Minimum requirements not met
+      shortcut: min
+      category: invalid
+      message: |-
+        *Thank you for your report!* 
+        However, this issue is {color:#FF5722}*Invalid*{color}.
+
+        Your computer does not meet the [minimum system requirements|https://help.minecraft.net/hc/en-us/articles/360038937032-Dungeons-Minimum-Specifications-for-Gameplay] to run Minecraft Legends.
```

Comment:
I'm assuming there's no Legends-specific one?

Reply:
Currently trying to find that out, at the very least I haven't found anything official.

Reply:
I'll leave this as is for now, we can always change it later.